minitests/roi_harness/runme.tcl

```diff
@@ -360,6 +360,48 @@ proc node2wire {node} {
     return $wire
 }
 
+proc write_grid_roi {fp} {
```

Comment:
This completely ignores XRAY_ROI_GRID in favor of XRAY_ROI. Wouldn't it be better to go the other direction and assume that we want all primitives within XRAY_ROI_GRID?

Reply:
The current runme.tcl code doesn't obey XRAY_ROI_GRID at all, so I'm not sure how we would leverage XRAY_ROI_GRID.

Reply:
Hmm but isn't our environment guaranteeing they are related?

Reply:
Sure!  Done